### DISCUSSION TITLE
Attention syntax works for flux too?

### DISCUSSION THREAD

**Discussion by ZeroCool22** [lllyasviel/stable-diffusion-webui-forge#1332] [Q&A]
I mean:

**(realistic style:1.2)** it works with FLUX too or only for SDXL?

**Comment by HMRMike** [comment]
There's definitely an effect, but I guess up to a limit of sorts, depending on content. Some LORA would be more beneficial to get the style you aim for.
photograph of a colorful ball on a beach
![00010-4022993114](https://github.com/user-attachments/assets/f547fffe-a4ae-48c0-93c2-c5da9a86389c)

(colorful:0.1)
![00013-4022993114](https://github.com/user-attachments/assets/59325478-7a31-465c-8b91-80450c9848bb)

(colorful:0.3)
![00015-4022993114](https://github.com/user-attachments/assets/33d117cb-b3d9-4191-834d-a2ae465dbcb9)

(colorful:2) wasn't very different from the baseline. But even extreme values don't seem to cause things to break catastrophically like SDXL behaves
![00012-4022993114](https://github.com/user-attachments/assets/3339f115-c4f8-43f6-b894-20f5031ab8b7)
